.github/CODEOWNERS

```diff
@@ -58,4 +58,6 @@
 
 /src/aks-preview/ @zqingqing1
 
-/src/sqlvm-preview/ @yareyes
\ No newline at end of file
+/src/sqlvm-preview/ @yareyes
+
+/src/anf-preview/ @williexu
```

Comment:
Can you change this to reference whoever on your team will be the prime codeowner/reviewer?